packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -161,6 +162,23 @@ export abstract class SharedObjectCore<
 		const { opProcessingHelper, callbacksHelper } = this.setUpSampledTelemetryHelpers();
 		this.opProcessingHelper = opProcessingHelper;
 		this.callbacksHelper = callbacksHelper;
+		this.processMessagesHelper =
+			this.processMessagesCore === undefined
+				? (messagesCollection: IRuntimeMessageCollection) =>
+						processCoreHelper(messagesCollection, this.processCore.bind(this))
```

Comment:
should this be calling process, instead of process core?

Reply:
Yes, I totally missed that. No test failed because of not emitting the op events!